types/sharepoint/index.d.ts

```diff
@@ -11520,6 +11660,41 @@ declare namespace Srch {
         viewDuplicates(docId: number): void;
         /** Returns true if the specified table has results and is enabled to be shown by this control */
         shouldShowTable(resultTable: Microsoft.SharePoint.Client.Search.Query.ResultTable): boolean;
+
+        setAriaCollapsed: (element: HTMLElement) => void;
+        get_showQuerySuggestions: () => boolean;
+        get_showNavigation: () => boolean;
+        get_initialPrompt: () => string;
+        get_currentTerm: () => string;
+        handleClickOnCategoryLink: (id: string) => void;
+        get_advancedSearchPageAddress: () => string;
+        get_emptyRefinementMessageId: () => string;
+        get_selectedRefinementControls: () => CSRRefinementControl[];
+        activate: (prompt: string,
+            searchBoxId: string, searchBoxDivId: string,
+            navButtonId: string, suggestionsListId: string,
+            navListId: string, searchBoxLinkId: string,
+            searchBoxProgressClass: string, searchBoxPromptClass: string) => void;
+    }
+
+    interface RefinementCategory {
+        k: boolean; // Use KQL,
+        m: { [key: string]: string } | null; // token to display value map
```

Comment:
please use comments with two asterisks and place them before the property, so that those comments are visible in the intellisense, for example like this:
```ts
interface RefinementCategory {
        /** use KQL */
        k: boolean;
        /** token to display value map */
        m: { [key: string]: string } | null;
        // ....
}
```